docs/apache-airflow-providers-amazon/auth-manager/setup/identity-center.rst

```diff
@@ -0,0 +1,142 @@
+
+The CLI command is not compatible with AWS account managed through AWS organizations.
+If your AWS account is managed through AWS Organizations, please follow the
+:ref:`manual configuration <identity_center_manual_configuration>`.
+
+In order to create all resources needed by the AWS auth manager, you can use the CLI command vended as part of the AWS auth manager.
+In order to use it, you first need to set the AWS auth manager as auth manager in Airflow config.
+See :doc:`how to set AWS auth manager as auth manager </auth-manager/setup/config>`.
+
+Once done, to create the resources, please run the following command:
+
+.. code-block:: bash
+
+  airflow aws-auth-manager init-identity-center
+
+The CLI command should exit successfully with the message. ::
+
+  AWS IAM Identity Center resources created successfully.
+
+If the CLI command exited with an error, please look carefully at the CLI command output to understand which resource(s)
+have been successfully created and which have not. The resource(s) which have not been successfully created need to be
+:ref:`created manually <identity_center_manual_configuration>`.
+
+If the error message below is raised, please create the AWS IAM Identity Center application through the console
+following :ref:`these instructions <identity_center_manual_configuration_application>`. ::
+
+   Creation of SAML applications is only supported in AWS console today. Please create the application through the console.
+
+.. _identity_center_manual_configuration:
+
+Create resources manually
+-------------------------
+
+Create the instance
+~~~~~~~~~~~~~~~~~~~
+
+Please follow `AWS documentation <https://docs.aws.amazon.com/singlesignon/latest/userguide/identity-center-instances.html>`_
+to create the AWS IAM Identity Center instance.
+
+.. _identity_center_manual_configuration_application:
+
+Create the application
+~~~~~~~~~~~~~~~~~~~~~~
+
+Please follow the instructions below to create the AWS IAM Identity Center application.
+
+1. Open the `IAM Identity Center console <https://console.aws.amazon.com/singlesignon>`_.
+2. Choose **Applications**.
+3. Choose the **Customer managed** tab.
+4. Choose **Add application**.
+5. On the **Select application type** page, under **Setup preference**, choose **I have an application I want to set up**.
+6. Under **Application type**, choose **SAML 2.0**.
+7. Choose **Next**.
+8. On the **Configure application** page, under **Configure application**, enter a **Display name** for the application, such as ``Airflow``. Then, enter a Description.
+9. Under **IAM Identity Center metadata**, copy the address of the **IAM Identity Center SAML metadata file**.
+
+.. note::
+
+  You will need to set this address in Airflow configuration later.
+
+10. Under **Application metadata**, choose **Manually type your metadata values**. Then, provide the **Application ACS URL** and **Application SAML audience** as follows:
+
+   * **Application ACS URL**: ``<airflow_endpoint>/login_callback``
```

Comment:
Is `<airflow_endpoint>` a place holder that the individual following this guide should know to fill in here? Or is that literally the text you should type in?

Reply:
> Maybe I should mention it and say they should replace it with their actual endpoint?

Yeah I think that'd be really helpful. A sentence before this code block describing what that thing is and where the user can get the value for it.

Reply:
Done